packages/garbo/src/tasks/post/index.ts

```diff
@@ -339,11 +343,34 @@ function handleDrenchedInLava(): GarboPostTask {
   };
 }
 
+let bestAbortFreeRun: Item | null = null;
+function getBestAbortFreeRun(): Item {
+  if (bestAbortFreeRun === null) {
+    const bestFreeRun = maxBy(unlimitedFreeRunList, getAcquirePrice, true);
+    bestAbortFreeRun =
+      getAcquirePrice(bestFreeRun) < get("valueOfAdventure")
+        ? bestFreeRun
+        : $item.none;
+  }
+  return bestAbortFreeRun;
+}
+
+function acquireAbortFreeRun(): GarboPostTask {
+  return {
+    name: "Acquire Best Free Run in Case of Abort",
+    completed: () =>
+      getBestAbortFreeRun() === $item`none` || have(getBestAbortFreeRun()),
+    do: () =>
+      acquire(1, getBestAbortFreeRun(), getAcquirePrice(getBestAbortFreeRun())),
```

Comment:
pass in value of adventure as the third arg and make sure you pass the "pwease don't abort" arg too

after that we're good